Updated pt and pt-BR languages

tiagofrancafernandes opened the pull request:
## Description

Updated files:
- `lang/lang.csv`
- `lang/pt-BR/lang.php`
- `lang/pt/lang.php`
- `views/users/form.blade.php`
- `config/twill.php`

tiagofrancafernandes committed:
Update lang/pt/lang.php

tiagofrancafernandes committed:
Update lang/pt-BR/lang.php

tiagofrancafernandes committed:
Update lang.csv

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/area17/twill?pullRequest=2524) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/area17/twill?pullRequest=2524) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: tiagofrancafernandes<br/>:white_check_mark: ifox<br/>:x: Tiago França<br/><hr/>**Tiago França** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/area17/twill?pullRequest=2524) it.</sub>

tiagofrancafernandes committed:
Update twill.php

tiagofrancafernandes committed:
Update form.blade.php

tiagofrancafernandes committed:
Update pt-BR/lang.php

tiagofrancafernandes committed:
Update pt/lang.php

Commit:
added pt-BR lang

On branch 3.x-tgo
	modified:   config/twill.php
	modified:   views/users/form.blade.php

Commit:
Merge branch '3.x-tgo' of github.com:tiagofrancafernandes/twill-laravel-cms into 3.x-tgo

ifox commented on views/users/form.blade.php:
```diff
@@ -12,6 +12,7 @@
 
     <x-twill::input
         name="email"
+        :placeholder="twillTrans('twill::lang.user-management.email')"
```
Thanks @tiagofrancafernandes! I'm not sure I understand why a placeholder is needed here.

ifox committed:
Merge branch '3.x' into 3.x-tgo
